src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ManageContent/ManageContentPageService.cs

```diff
@@ -74,24 +82,45 @@ public class ManageContentPageService : IManageContentPageService
                 });
         }
 
-        private static IQueryable<Release> HydrateReleaseForReleaseViewModel(IQueryable<Release> values)
+        private async Task<Release> HydrateReleaseForReleaseViewModel(Release release)
         {
-            return values
-                .Include(r => r.Publication)
-                .ThenInclude(publication => publication.Contact)
-                .Include(r => r.Publication)
-                .ThenInclude(publication => publication.Releases)
-                .Include(r => r.Publication)
-                .ThenInclude(publication => publication.LegacyReleases)
-                .Include(r => r.Publication)
-                .ThenInclude(publication => publication.Topic.Theme)
-                .Include(r => r.Type)
-                .Include(r => r.Content)
-                .ThenInclude(join => join.ContentSection)
-                .ThenInclude(section => section.Content)
-                .ThenInclude(content => content.Comments)
-                .ThenInclude(comment => comment.CreatedBy)
-                .Include(r => r.Updates);
+            release.Updates = _contentDbContext.Update
+                .Where(u => u.ReleaseId == release.Id)
+                .ToList();
```

Comment:
There's a couple of places that are ToList here, rather than ToListAsync. I would make everything ToListAsync.